code.test.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const assert = require('assert');
+eval(fs.readFileSync('code.js')+'');
+
+function floydWarshall(graph) { 
```

Comment:
How do you know this implementation here is correct?

Reply:
I would assume this implementation is correct as following it through with your psuedocode within the readme.md it does properly follow each of the steps from my understanding.
1. creates a VxV matrix of infinites
2. makes the diagonals zeros to account for the the locations where verticies overlaps with themselves
3. adding the weights of each edge to the proper position
4. performs the triple loop check where if the current distance/weight within the position is greater than another alternative path, replace the old weight with the smaller one.

Reply:
...but it's a bit weird to test the code you've written with other code you've written. It would be better to check a few cases explicitly here.

Reply:
So the function that i used in the testing was directly just copy and pasted from a website that was explaining and talking about the floyd warshall, i did have the website linked at the bottom of the function block. But i can do the explicit case testing regardless.